examples/logit_ratio.py

```diff
@@ -75,8 +77,8 @@
     model_variables=[cascade_model_variables],
     prior_masks={"intercept": [1.0],
                  "idr_lagged": [1.0]*idr_variable.size,
-                 "time_id": [1.0]*time_variable.size},
-    level_masks=[1.0, 1.0, 10.0],
+                 "time_id": [1.0]*(time_variable.size - 1) + [0.1]},
```

Comment:
Test it out different prior strength on different spline bases, this will make the prior on the last spline basis tighter, and the tail follow the prior better.